rpc_jobs/rpc_aio.yml

```diff
@@ -205,11 +205,14 @@
                 tempest.tempest()
                 holland.holland()
               }}
+              currentBuild.result = "SUCCESS"
             }} catch (e) {{
               print(e)
               throw e
             }} finally {{
-              common.submit_ara()
+              if(currentBuild.result != "SUCCESS"){{
+                common.submit_ara()
```

Comment:
This line could have been moved into the `catch` section which would save having to set the result etc.